### PR TITLE
[Localization] various zh_cn translation fix (7-1)

### PR DESCRIPTION
**Pull request by 40chyan**
## What are the changes?
fixing text translation, visual display issues, typos, grammar issues, etc.

## Why am I doing these changes?
better localization

## What did change?
**in-battle texts**
add missing translation
bunch of unnecessary spaces for zh_cn deleted, make sentences more fluent, we don't have that many spaces in Chinese sentences.do linebreak as the original pokemon games
image

**items**
fix grammar and linebreak
fix two item description that have multiple "\n" which causes text not being fully displayed, also shorten those.
image

**bgm**
add translation for new one and original musics

**abilities**
remove the requirement of no item to be held from thief and ball fetch abilities, because they don't exist

**dialogue**
fixing elite raihan's dialogue based on original texts
image

**splash text**
fix translation to make it more colloquial, also fullwidth "！"
btw, the splash texts are OLD, and apparently cn people can't understand most of them
I suppose we should make a big change on splashes some day, or maybe different texts based on languages.

**starter ui**
toggel IV: "切换个体" to "显示" based on language habits

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/zh_CN/battle.ts [hunk 1]
```diff
@@ -4,132 +4,132 @@ export const battle: SimpleTranslationEntries = {
...
+  "hitResultNoEffect": "对{{pokemonName}}没有效果！！",
   "hitResultOneHitKO": "一击必杀！",
   "attackFailed": "但是失败了！",
-  "attackHitsCount": "击中 {{count}} 次！",
+  "attackHitsCount": "击中{{count}}次！",
   "rewardGain": "你获得了\n{{modifierName}}！",
-  "expGain": "{{pokemonName}} 获得了 {{exp}} 经验值！",
-  "levelUp": "{{pokemonName}} 升级到 Lv.{{level}}！",
+  "expGain": "{{pokemonName}}获得了 {{exp}} 点经验值！",
+  "levelUp": "{{pokemonName}}升级到 {{level}}级！",
```
Comment (40chyan) [inline src/locales/zh_CN/battle.ts:31]:
```suggestion
  "levelUp": "{{pokemonName}}升级到 Lv.{{level}}！",
```

Comment (RimKnight) [inline src/locales/zh_CN/battle.ts]:
```suggestion
  "levelUp": "{{pokemonName}} 升级到 Lv.{{level}}！",
```

Comment (40chyan) [reply src/locales/zh_CN/battle.ts]: fixed already

File: src/locales/zh_CN/modifier-type.ts [hunk 2]
```diff
@@ -42,11 +42,11 @@ export const modifierType: ModifierTypeTranslationEntries = {
       }
     },
     "PokemonPpUpModifierType": {
-      description: "为一只宝可梦的一个招式永久增加{{upPoints}}点\nPP每5点当前最大PP (最多3点)。",
+      description: "为一只宝可梦的一个招式的PP永久增加{{upPoints}}20%的原最大PP值 (最多3次)。",
```
Comment (RimKnight) [inline src/locales/zh_CN/modifier-type.ts:45]:
```suggestion
      description: "选择一只宝可梦的一个招式使用\n使其PP最大值提升基础的20％ (最多3次)。",
```

File: src/locales/zh_CN/modifier-type.ts [hunk 3]
```diff
@@ -254,7 +254,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
     "x_sp_def": "特防强化",
     "x_speed": "速度强化",
     "x_accuracy": "命中强化",
-    "dire_hit": "要害攻击",
+    "dire_hit": "会心强化",
```
Comment (RimKnight) [inline src/locales/zh_CN/modifier-type.ts:257]:
```suggestion
    "dire_hit": "要害攻击",
```

**Review by RimKnight — APPROVED** [review]

**Review by RimKnight — APPROVED** [review]
No other issues. Good job!

**Review by RimKnight — APPROVED** [review]

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**